t/mojolicious/exception_lite_app.t

```diff
@@ -126,149 +126,199 @@ get '/dead_helper';
-  ->content_like(qr/Page not found/);
-
-# Missing template with custom rendering
-$t->get_ok('/missing_template.txt')->status_is(404)->content_type_is('text/plain;charset=UTF-8')
-  ->content_is('Missing template, whatever.');
-
-# Missing template (failed rendering)
-$t->get_ok('/missing_template/too')->status_is(404)->header_is('X-Not-Found' => 1)
-  ->content_type_is('text/html;charset=UTF-8')->content_like(qr/Page not found/);
-
-# Missing helper (correct context)
-$t->get_ok('/missing_helper')->status_is(500)->content_type_is('text/html;charset=UTF-8')
-  ->content_like(qr/Server error/)->content_like(qr/shift-&gt;missing_helper/);
-
-# Reuse exception
-ok !$exception, 'no exception';
-ok !$snapshot,  'no snapshot';
-$t->get_ok('/reuse/exception')->status_is(500)->content_like(qr/Reusable exception/);
-isa_ok $exception, 'Mojo::Exception',      'right exception';
-like $exception,   qr/Reusable exception/, 'right message';
-is $snapshot->{foo}, 'bar', 'right snapshot value';
-ok !$snapshot->{exception}, 'no exception in snapshot';
-
-# Bundled static files
-$t->get_ok('/mojo/jquery/jquery.js')->status_is(200)->content_type_is('application/javascript');
-$t->get_ok('/mojo/prettify/run_prettify.js')->status_is(200)->content_type_is('application/javascript');
-$t->get_ok('/mojo/prettify/prettify-mojo-dark.css')->status_is(200)->content_type_is('text/css');
-$t->get_ok('/mojo/failraptor.png')->status_is(200)->content_type_is('image/png');
-$t->get_ok('/mojo/logo.png')->status_is(200)->content_type_is('image/png');
-$t->get_ok('/mojo/logo-black.png')->status_is(200)->content_type_is('image/png');
-$t->get_ok('/mojo/logo-black-2x.png')->status_is(200)->content_type_is('image/png');
-$t->get_ok('/mojo/logo-white.png')->status_is(200)->content_type_is('image/png');
-$t->get_ok('/mojo/logo-white-2x.png')->status_is(200)->content_type_is('image/png');
-$t->get_ok('/mojo/noraptor.png')->status_is(200)->content_type_is('image/png');
-$t->get_ok('/mojo/notfound.png')->status_is(200)->content_type_is('image/png');
-$t->get_ok('/mojo/pinstripe-dark.png')->status_is(200)->content_type_is('image/png');
-$t->get_ok('/mojo/pinstripe-light.png')->status_is(200)->content_type_is('image/png');
+subtest 'Dead included template' => sub {
+  $t->get_ok('/dead_included_template')->status_is(500)->content_like(qr/dead template!/)->content_like(qr/line 1/);
+};
+
+subtest 'Dead template with layout' => sub {
+  $t->get_ok('/dead_template_with_layout')->status_is(500)->content_like(qr/dead template with layout!/)
+    ->content_like(qr/line 2/)->content_unlike(qr/Green/);
+
+  like $log, qr/dead template with layout!/, 'right result';
+};
+
+subtest 'Dead action' => sub {
+  $t->get_ok('/dead_action')->status_is(500)->content_type_is('text/html;charset=UTF-8')
+    ->content_like(qr!get &#39;/dead_action&#39;!)->content_like(qr/dead action!/)
+    ->text_is('#error' => "dead action!\n");
+
```

Comment:
Same here.

Reply:
I take this is the same for the other `Dead XXX` subtests?

Reply:
And perhaps for the logger level subtests as well?

Reply:
I went ahead and just removed all extra newlines for those subtests...